plugins/PrivacyManager/SystemSettings.php

```diff
@@ -53,12 +61,35 @@ private function createTermsAndConditionUrlSetting()
         });
     }
 
-    private function createShowInEmbeddedWidgetsSetting()
+    private function createShowInEmbeddedWidgetsSetting(): SystemSetting
     {
         return $this->makeSetting('showInEmbeddedWidgets', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
             $field->title = Piwik::translate('PrivacyManager_ShowInEmbeddedWidgets');
             $field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
             $field->description = Piwik::translate('PrivacyManager_ShowInEmbeddedWidgetsDescription');
         });
     }
+
+    private function createForceCookielessTrackingSetting(): SystemConfigSetting
+    {
+        return $this->makeSettingManagedInConfigOnly('Tracker', 'force_cookieless_tracking', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
+            $field->title = Piwik::translate('PrivacyManager_ForceCookielessTracking');
+            $field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
+            $field->description = Piwik::translate('PrivacyManager_ForceCookielessTrackingDescription');
+        });
+    }
+
+    public function save()
+    {
+        parent::save();
+
+        try {
+            if (Manager::getInstance()->isPluginActivated('CustomJsTracker')) {
```

Comment:
wonder if we should call the method `updateTracker` on the plugin instance directly? Or have a new event like `CustomJsTracker.updateTracker` to trigger this logic?

Reply:
sure, why not. will change that...